net-p2p/resilio-sync-bin/resilio-sync-bin-2.5.6.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit pax-utils systemd tmpfiles unpacker user
+
+MY_PN="${PN/-bin/}"
```

Comment:
Why name it `-bin` just to make the ebuild harder? The `-bin` suffix is generally required only if there's a non`-bin` alternative (i.e. firefox vs firefox-bin).